nematode/GPSFix.cpp

```diff
@@ -244,21 +247,14 @@ GPSFix::~GPSFix()
 }
 
 // Returns the duration since the Host has received information
-seconds GPSFix::timeSinceLastUpdate()
+milliseconds GPSFix::timeSinceLastUpdate()
 {
-  time_t now      = time(NULL);
-  struct tm stamp = {0};
-
-  stamp.tm_hour = timestamp.hour;
-  stamp.tm_min  = timestamp.min;
-  stamp.tm_sec  = (int)timestamp.sec;
-  stamp.tm_year = timestamp.year - 1900;
-  stamp.tm_mon  = timestamp.month - 1;
-  stamp.tm_mday = timestamp.day;
-
-  time_t then   = mktime(&stamp);
-  uint64_t secs = (uint64_t)difftime(now, then);
-  return seconds((uint64_t)secs);
+  auto time1 = std::chrono::system_clock::now();
+
+  auto time2 = timestamp.getTime();
+  auto diff =
+      time1.time_since_epoch().count() - time2.time_since_epoch().count();
```

Comment:
I think it's safer to use `duration_cast<milliseconds>(time1 - time2)` because `count` may not be the same `millisecond` `Period` in `time1` and `time2`.